### PR TITLE
June podcast

### PR DESCRIPTION
**Pull request by edunham**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL approved by Dev Advocacy
- [ ] The content has been run through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist
- [ ] Images are compressed appropriately
- [ ] Social image previews well on Twitter and LinkedIn
- [ ] Tech review request (developer advocate or domain expert)
- [ ] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+What's the automation journey, and how does it relate to organizational maturity? How can you scale your work with automation while avoiding the pitfalls of premature optimization? 
+
+If you're ready to start automating Okta administration tasks, where should you start? 
+
+## Episode Links
+
+{% youtube LilghHdW18Q %} 
+
+Listen to the audio version on Casted [here](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f/59f5cd1e).
```
Comment (alisaduncan) [inline _source/_posts/2023-06-30-podcast-approaching-identity-automation.md]:
```suggestion
Listen to the [audio version on Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f/59f5cd1e). 
```

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+If you're ready to start automating Okta administration tasks, where should you start? 
+
+## Episode Links
+
+{% youtube LilghHdW18Q %} 
+
+Listen to the audio version on Casted [here](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f/59f5cd1e). 
+
+Where are you in your own automation journey? What are your next steps to enhance automation? Let us know in the comments below!
```
Comment (alisaduncan) [inline _source/_posts/2023-06-30-podcast-approaching-identity-automation.md:27]: Probably should add the CTA to follow OktaDev on Twitter and subscribe on YouTube (can they subscribe on Casted??) to get notified about the next episode or other exciting content.

Also do you have resources that people may want to read more about?

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+author: edunham
+by: advocate
+communities: [devops,security]
+description: "Listen and learn about the automation journey"
+tags: [podcast, automation, enterprise]
+image: blog/identity-developer-podcast/podcast.jpg
+type: awareness
+---
+
+# Approaching Automation
```
Comment (alisaduncan) [inline _source/_posts/2023-06-30-podcast-approaching-identity-automation.md]:
```suggestion
# Approaching automation in identity
```

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Product Manager Jeff Taylor joins us for the first time to discuss the principles behind successful automation. 
+
+What's the automation journey, and how does it relate to organizational maturity? How can you scale your work with automation while avoiding the pitfalls of premature optimization? 
+
+If you're ready to start automating Okta administration tasks, where should you start? 
+
+## Episode Links
```
Comment (alisaduncan) [inline _source/_posts/2023-06-30-podcast-approaching-identity-automation.md]:
```suggestion
## Find the OktaDev Workforce podcast on approaching identity automation
```

trying to reinforce keywords

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+layout: blog_post
+title: "June Podcast: Approaching Automation with Jeff Taylor"
+author: edunham
+by: advocate
+communities: [devops,security]
+description: "Listen and learn about the automation journey"
```
Comment (alisaduncan) [inline _source/_posts/2023-06-30-podcast-approaching-identity-automation.md]:
```suggestion
description: "Listen and learn about the identity automation journey"
```

**Review by alisaduncan — COMMENTED** [review]
A few comments

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md (hunk 3 above)
Comment (alisaduncan) [reply _source/_posts/2023-06-30-podcast-approaching-identity-automation.md]: actually I just realized this is a h1. this should be deleted (always start with h2) and we don't need a header here

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md (hunk 2 above)
Comment (alisaduncan) [reply _source/_posts/2023-06-30-podcast-approaching-identity-automation.md:27]: @edunham I think adding the same (or similar) CTA you have for July podcast here will be 👍🏼 . Then let's ship it. :shipit:

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+If you're ready to start automating Okta administration tasks, where should you start? 
+
+## Find the OktaDev Workforce podcast on approaching identity automation
+
+{% youtube LilghHdW18Q %} 
+
+Listen to the [audio version on Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f/59f5cd1e). 
+
+Where are you in your own automation journey? What are your next steps to enhance automation? Let us know in the comments below!
```
Comment (edunham) [inline _source/_posts/2023-06-30-podcast-approaching-identity-automation.md:27]:
```suggestion
Where are you in your own automation journey? What are your next steps to enhance automation? Let us know in the comments below!

Follow OktaDev on [YouTube](https://www.youtube.com/@OktaDev) and [Twitter](https://twitter.com/oktadev) to learn more about automating identity!
```

File: _source/_posts/2023-06-30-podcast-approaching-identity-automation.md (hunk 2 above)
Comment (edunham) [reply _source/_posts/2023-06-30-podcast-approaching-identity-automation.md:27]: twitter/youtube added here, cta PR'd for July :)

**Review by alisaduncan — APPROVED** [review]
:shipit:

**Pull request merged**